synapse_data_warehouse/synapse_raw/tables/V2.15.0__add_columns_to_cert.sql

```diff
@@ -0,0 +1,4 @@
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP
```

Comment:
Why is `accountadmin` not used when adding these columns?

Reply:
Great question - by default, it uses SYSADMIN roll to modify our table objects.